Neos.Flow/Documentation/TheDefinitiveGuide/PartII/Configuration.rst

```diff
@@ -184,12 +184,12 @@ Some specific flow behaviour can also be configured with a couple of environment
 ============================	==================================================
 Variable						Description
 ============================	==================================================
-FLOW_ROOTPATH					Can be used to override the path to the Flow root.
-FLOW_CONTEXT					Use to set the flow context (see above).
-FLOW_PATH_TEMPORARY_BASE		Can be used to set a path for temporary data.
-FLOW_LOCKHOLDINGPAGE			Use to specify the html page shown when the site is locked.
-								This is relative to the Packages directory.
-FLOW_ONLY_COMPOSER_LOADER		Set to true (1) to only use composer autoloader.
+FLOW_ROOTPATH									Can be used to override the path to the Flow root.
```

Comment:
The indentation around here is wonky (now).

Reply:
I think this is a Github issue, in the IDE it looks fine now:
![image](https://user-images.githubusercontent.com/307571/100470347-61fd6e00-30d8-11eb-96b6-e72d0e43b9ba.png)

Reply:
otoh.. that probably depends on the IDE configuration for tab width..

Reply:
Yes, spaces FTW…

Reply:
But those used to be tabs (see https://raw.githubusercontent.com/neos/flow-development-collection/6.0/Neos.Flow/Documentation/TheDefinitiveGuide/PartII/Configuration.rst). Should I change that?

Reply:
Well, either leave them untouched or fix it for real. :)

Reply:
I'll create "proper" tables now

Reply:
Done